libraries/AC_AutoTune/AC_AutoTune.h

```diff
@@ -210,8 +211,8 @@ class AC_AutoTune
         RD_DOWN = 1,              // rate D is being tuned down
         RP_UP = 2,                // rate P is being tuned up
         RFF_UP = 3,               // rate FF is being tuned up
-        SP_UP = 4,                // angle P is being tuned up
-        SP_DOWN = 5,              // angle P is being tuned down
+        SP_DOWN = 4,              // angle P is being tuned down
+        SP_UP = 5,                // angle P is being tuned up
```

Comment:
@lthall You shouldn't change the numbering here if you want to change the sequence of the tuning.  You change that in the Set_Tune_Sequence method in AC_AutoTune_Multi.h

Reply:
I changed it here so it matches the ordering. This is really just to make it easier to read and review the logs.